app/models/preview_card.rb

```diff
@@ -55,7 +55,9 @@ class PreviewCard < ApplicationRecord
 
   has_one :trend, class_name: 'PreviewCardTrend', inverse_of: :preview_card, dependent: :destroy
 
-  has_attached_file :image, processors: [:lazy_thumbnail, :blurhash_transcoder], styles: ->(f) { image_styles(f) }, validate_media_type: false
+  has_attached_file :image, processors: [ENV['MASTODON_USE_LIBVIPS'] == true ? :vips_lazy_thumbnail : :thumbnail, :blurhash_transcoder], styles: lambda { |f|
```

Comment:
I would prefer if we have a global option somewhere global (on the `Rails` object?), initialized at startup, and we check the value here